projects/json-c/tokener_parse_ex_fuzzer.cc

```diff
@@ -0,0 +1,10 @@
+#include <stdint.h>
+
+#include <json.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const char *data, size_t size) {
+  json_tokener *tok = json_tokener_new();
+  json_tokener_parse_ex(tok, data, size);
```

Comment:
If you're doing any leak checking you should free any generated json_object returned here with json_object_put().

Reply:
thank you - I'll do this.

Reply:
That fixed my leaks - thanks.